packages/node/src/module.ts

```diff
@@ -9,52 +8,62 @@ function normalizeWindowsPath(path: string): string {
     .replace(/\\/g, '/'); // replace all `\` instances with `/`
 }
 
+// We cache this so we don't have to recompute it
+let basePath: string | undefined;
+
+function getBasePath(): string {
+  if (!basePath) {
+    const baseDir =
```

Comment:
l: This is not really related to this PR, but it would be nice to have a comment explaining what & why we actually do here - e.g. why use `require.main.filename` over `global.process.cwd()` etc. Feel free to ignore this, but as somebody with little to no historical context on this, this appears very magic to me 😅

Reply:
I have no idea other than this has been like this since 11+ years from raven-node:
https://github.com/getsentry/raven-node/blame/7019cb30efda40704105b7b510bc428ec0c1ee01/lib/utils.js#L87